spec/requests/product_drive_participants_requests_spec.rb

```diff
@@ -97,9 +97,11 @@
     end
 
     describe "GET #show" do
-      it "returns http success" do
-        get product_drive_participant_path(id: create(:product_drive_participant, organization: organization))
+      it "returns http success and displays comments" do
+        @participant = create(:product_drive_participant, organization: organization)
+        get product_drive_participant_path(id: @participant)
         expect(response).to be_successful
+        expect(response.body).to include(@participant.comment)
```

Comment:
Can we hardcode this to a string value when you create the participant? Otherwise, this could be hiding (say) the fact that the comment is an empty string.

Reply:
resolved by [b059001](https://github.com/rubyforgood/human-essentials/pull/4442/commits/b059001d18221d6b12b599c16f0d910dd5150c73)